toolkit/templates/forms/selection-buttons.html

```diff
@@ -8,9 +8,9 @@
 {% if error %}
   <div class="validation-wrapper">
 {% endif %}
-  <fieldset class="question first-question" id="{{ name }}">
+  <fieldset class="question first-question{% if inline or type=='boolean' %} inline{% endif %}" id="{{ name }}">
     <legend>
-      <span class="question-heading{% if hint or question_advice %} question-heading-with-hint{% endif %}">
+      <span class="question-heading{% if hint_underneath %} question-heading-with-hint-underneath{% endif %}">
```

Comment:
The work done on vertical spacing (mentioned elsewhere on this PR) removed the need for `question-headings-with-hint`.